app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -46,6 +46,7 @@ def start
     image = target_entity
     return queue_signal(:abort_job, "no image found", "error") unless image
     return queue_signal(:abort_job, "cannot analyze non docker images", "error") unless image.docker_id
+    # return queue_signal(:abort_job, "role-missing", "error") unless CURRENT_SERVER.active_roles.include?("smartproxy")
```

Comment:
Thanks. We have a reproducible bug where the job is picked up by the development server
if it has the role smartstate but does not have smartproxy. It fails with a timeout.

hmm currently I think we queue the job with out any [role](https://github.com/moolitayer/manageiq/blob/2cc89dd3302134e961a3e7997e6945ade0c9b2fc/app/models/manageiq/providers/kubernetes/container_manager_mixin.rb#L119)

Would it be possible to queue it so that it would be picked up only by a server that has both smartproxy & smartstate?